tap_linkedin_ads/client.py

```diff
@@ -27,65 +26,99 @@ class LinkedInUnauthorizedError(LinkedInError):
     pass
 
 
-class LinkedInPaymentRequiredError(LinkedInError):
+class LinkedInMethodNotAllowed(LinkedInError):
     pass
 
 
 class LinkedInNotFoundError(LinkedInError):
     pass
 
-
-class LinkedInConflictError(LinkedInError):
+class LinkedInForbiddenError(LinkedInError):
     pass
 
+class LinkedInLengthRequired(LinkedInError):
+    pass
 
-class LinkedInForbiddenError(LinkedInError):
+class LinkedInRateLimitExceeeded(Server429Error):
     pass
 
+class LinkedInInternalServiceError(Server5xxError):
+    pass
 
-class LinkedInInternalServiceError(LinkedInError):
+class LinkedInGatewayTimeoutError(Server5xxError):
     pass
 
 
 ERROR_CODE_EXCEPTION_MAPPING = {
-    400: LinkedInBadRequestError,
-    401: LinkedInUnauthorizedError,
-    402: LinkedInPaymentRequiredError,
-    403: LinkedInForbiddenError,
-    404: LinkedInNotFoundError,
-    409: LinkedInForbiddenError,
-    500: LinkedInInternalServiceError}
-
-
-def get_exception_for_error_code(error_code):
-    return ERROR_CODE_EXCEPTION_MAPPING.get(error_code, LinkedInError)
+    400: {
+        "raise_exception": LinkedInBadRequestError,
+        "message": "The request is missing or has a bad parameter."
+    },
+    401: {
+        "raise_exception": LinkedInUnauthorizedError,
+        "message": "Invalid authorization credentials."
+    },
+    403: {
+        "raise_exception": LinkedInForbiddenError,
+        "message": "User doesn't have permission to access the resource."
```

Comment:
@harshpatel4crest Update the message to "User does not have permission to access the resource."

Reply:
Done